src/js/components/SchemaForm.js

```diff
@@ -107,7 +107,17 @@ class SchemaForm extends mixin(StoreMixin) {
       propID,
       newDefinition
     );
-    newDefinition.push(
+
+    let deleteButtonTop = Object.values(definition.itemShapes || {})
```

Comment:
I tried that but it seems to get stripped out or put on a different definition. Do you think this could have issues?